src/Xendit.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/**
+ * Xendit.php
+ * php version 7.2.0
+ *
+ * @category Class
+ * @package  Xendit
+ * @author   Ellen <ellen@xendit.co>
+ * @license  https://opensource.org/licenses/MIT MIT License
+ * @link     https://api.xendit.co
+ */
+
+namespace Xendit;
+
+/**
+ * Class Xendit
+ *
+ * @category Class
+ * @package  Xendit
+ * @author   Ellen <ellen@xendit.co>
+ * @license  https://opensource.org/licenses/MIT MIT License
+ * @link     https://api.xendit.co
+ */
+class Xendit
+{
+    public static $apiKey;
+
+    public static $apiBase = 'https://api.xendit.co';
+
+    public static $libVersion;
```

Comment:
@ellennugroho I'm not sure where this value being set. Is the static method `getLibVersion` being called **everytime** something is trying to retrieve this property? If so, I would advise you to memoize the value

Reply:
Yeah, previous implementation calls `getLibVersion` everytime. I already redid it on the newer commits